config/routes.rb

```diff
@@ -5,7 +5,9 @@
     mount LetterOpenerWeb::Engine, at: "/letter_opener"
   end
 
-  devise_for :personalities
+  devise_for :personalities, controllers: {
+      sessions: "admin/devise/sessions"
```

Comment:
__[RuboCop]__

Layout/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.


<!-- Comment by [SideCI](https://sideci.com) -->